corehq/apps/case_importer/static/case_importer/js/import_history.js

```diff
@@ -19,7 +19,25 @@ hqDefine('case_importer/js/import_history.js', function () {
             // this prevents some jumpiness when not necessary
             // and is particularly bad if you're in the middle of editing a comment
             return !(_.chain(self.case_uploads()).pluck('upload_id').isEqual(_(data).pluck('upload_id')).value() &&
-                _.chain(self.case_uploads()).pluck('task_status').isEqual(_(data).pluck('task_status')).value());
+                _.chain(self.case_uploads()).pluck('task_status').map(function (task_status) {return task_status()})
+                    .isEqual(_(data).pluck('task_status').map(function (task_status) {return task_status()})).value());
```

Comment:
Thanks, very fair. I definitely let this line grow out of control. Fix coming right up!